src/textviewer.h

```diff
@@ -78,6 +78,7 @@ private slots:
   void modified();
   void patternChanged(QString newPattern);
   void findNext();
+  void showWhitespace_changed(int state);
```

Comment:
The other slots are full camel-case, could this be changed to `showWhitespaceChanged`?